packages/contracts-bedrock/src/cannon/libraries/MIPSSyscalls.sol

```diff
@@ -25,26 +54,37 @@ library MIPSSyscalls {
 
     uint32 internal constant EBADF = 0x9;
     uint32 internal constant EINVAL = 0x16;
+    uint32 internal constant EAGAIN = 0xb;
+    uint32 internal constant ETIMEDOUT = 110;
```

Comment:
Where are you pulling the value for this error?  I'm seeing 0x91 [here](https://github.com/golang/go/blob/d8392e69973a64d96534d544d1f8ac2defc1bc64/src/syscall/zerrors_linux_mips.go#L1592C1-L1592C31)

Reply:
Oh interesting. The errno values are different across architectures. I didn't know this. Will fix